### ISSUE TITLE
Peptide specific imd_anova count fix

### ISSUE THREAD

**Issue by rarichardson92** [pmartR/pmartR#109]
Applying imd_anova on non-peptide data throws an error; seems to look for "Peptide" to no avail. Possible fix in slightly adjusting calls as well as changing to more general edata cname? Reproducible code and error below:

```
lipid_object       <- pmartRdata::lipid_object
lipid_object       <- pmartR::group_designation(lipid_object, "Condition")
lipid_object2      <- pmartR::edata_transform(lipid_object, "log2")

lipid_object2 <- pmartR::applyFilt(pmartR::imdanova_filter(lipid_object2),
                                   lipid_object2,
                                   min_nonmiss_anova = 2,
                                   min_nonmiss_gtest = 3)
lipid_stats        <- pmartR::imd_anova(lipid_object2, 
                                        test_method = "combined")
```
--------

Error in Peptide %in% as.character(to_fix) : object 'Peptide' not found

--------

Code chunk in imd_anova:
```
#Get counts for rows in "anova_results" but not in "imd_counts"
  final_cnts <- Full_results[,grep("Count",colnames(Full_results))]
  msng_cnts <- which(is.na(rowSums(final_cnts))) 
  if(length(msng_cnts)>0){
    to_fix <- Full_results[msng_cnts,]$Peptide
    omicsData2 <- omicsData
    omicsData2$e_data <- omicsData$e_data%>%filter(Peptide%in%as.character(to_fix))
    new_cnts <- imd_test(omicsData = omicsData2, comparisons = NULL, pval_adjust = 'none', pval_thresh = pval_thresh)
    rm(omicsData2)
    #Replace the NA counts with the correct counts
    Full_results[msng_cnts,grep("Count",colnames(Full_results))] <- new_cnts$Results[,grep("Count",colnames(new_cnts$Results))]
  }
```

**Comment by rarichardson92** [comment]
Looking at the develop branch, it looks like this was already addressed by commit 376fa74467efaf4700fd2b7cab37bcc4823392a2  on the develop branch; should be fixed with update to master

**Comment by lmbramer** [comment]
Fix pulled into master
